.github/workflows/prod-deployment.yaml

```
@@ -0,0 +1,82 @@
+    - name: Debug TAG value
+      run: echo "TAG value - ${{ env.TAG }}"
+
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID_AL }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY_AL }}
+        aws-region: ${{ env.AWS_REGION }}
+        
+    - name: Setup Node Env
+      uses: actions/setup-node@v3
```

actionlint (1.7.7)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)